src/DataLib/fendl32B_retrofit/process_fendl3.2.py

```diff
@@ -8,5 +10,14 @@ def main():
 
     mt_dict = rxd.process_mt_data(rxd.load_mt_table('mt_table.csv'))
 
+    endf_path = 'files_for_tests/endf_test1.tendl'
+    pendf_path = 'files_for_tests/pendf_test1.pendf'
+
+    material_id, MTs = tpp.extract_endf_specs(endf_path)
+    njoy_template = groupr_tools.establish_static_template()
```

Comment:
I don't see this function anymore?  Don't we just have to import the template?

Reply:
Yep, you're right. That was a holdover from a previous version. I'll take that out.